src/components/calltoaction.js

```diff
@@ -20,7 +20,7 @@ const CallToAction = () => (
       <Row>
         <Col>
           <StaticImage src='../images/display-blk.png' alt='monitor' />
-          <a href="https://github.com/Verizon" target="_blank" rel="noreferrer">
+          <a rel="noreferrer" href="https://github.com/Verizon" target="_blank" >
```

Comment:
nit: I know this was changed by accident during our live-review, but I think this is better/more idiomatic with the href first?

Reply:
easy change sure